iocore/eventsystem/I_VConnection.h

```diff
@@ -149,7 +146,7 @@ typedef struct tsapi_vio *TSVIO;
   It is also a Continuation that is called back from processors.
 
 */
-class VConnection : public Continuation
+class VConnection : public Continuation, public PluginUserArgs<MAX_USER_ARGS_VCONN>
```

Comment:
Why is `PluginUserArgs` here instead of on `DummyConnection`?

Reply:
I don't think the VConnection in the ProxySession is a DummyVConnection ?

Reply:
Ok, as per your OOB suggested, hoisted up to NetVC